packages/animations/lib/src/utils/composite_animation_widget.dart

```diff
@@ -0,0 +1,253 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:animations/animations.dart';
+import 'package:flutter/material.dart';
+
+/// Signature for a function that creates a widget that builds a
+/// enter transition.
+///
+/// Used by [CompositeAnimationWidget]
+typedef _EnterTransitionBuilder = Widget Function(
+    BuildContext, Animation<double>, Widget);
+
+/// Signature for a function that creates a widget that builds a
+/// exit transition.
+///
+/// Used by [CompositeAnimationWidget]
+typedef _ExitTransitionBuilder = Widget Function(
```

Comment:
These typedefs are both the same. Just define it once with a more generic name, e.g. TransitionBuilder.

Reply:
got it